src/CBT.NuGet/version.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "https://raw.githubusercontent.com/AArnott/Nerdbank.GitVersioning/master/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "2.1",
+  "version": "2.2",
```

Comment:
Hopefully we remember to mention in the CBT.NuGet 2.2 release notes that CBT.Core 2.2 is required